oauth2_provider/views.py

```diff
@@ -11,7 +11,8 @@
 import provider.oauth2.forms
 import provider.scope
 from provider.oauth2.models import AccessToken
-from provider.oauth2.views import OAuthError, Capture, Redirect  # pylint: disable=unused-import
+from provider.oauth2.views import OAuthError, OAuth2AccessTokenMixin
+from provider.oauth2.views import Capture, Redirect  # pylint: disable=unused-import
```

Comment:
Can you fold these imports together? They're all coming from `provider.oauth2.views`.